.github/workflows/ci.yml

```diff
@@ -10,54 +9,67 @@ jobs:
-      uses: actions/checkout@v1
-    - name: Setup Go
-      uses: actions/setup-go@v1
-      with:
-        go-version: 1.13
-    - name: Install golint
-      run: |
-        go get -u golang.org/x/lint/golint
-    - name: Before build
-      run: |
-        echo Workflow trigger - ${{ github.event_name }}
-        ./hack/verify-gofmt.sh
-        ./hack/verify-govet.sh
-        ./hack/verify-golint.sh
-        go mod verify
-    - name: Build and test
-      run: |
-        make test
-        make
-    - name: Build container image
-      run: make container-image
-    - name: Push docker images
-      if: success() &&  github.event_name  == 'push'  && endsWith(github.ref ,'develop')
-      run: |
-        docker login -u ${{ secrets.DOCKER_USERNAME }} -p ${{ secrets.DOCKER_PASSWORD }}
-        docker push infracloudio/botkube:latest
-    - name: Install helm
-      if: success() &&  github.event_name  == 'push'  && endsWith(github.ref ,'develop')
-      run: |
-        curl https://raw.githubusercontent.com/kubernetes/helm/master/scripts/get > get_helm.sh
-        chmod +x get_helm.sh
-        ./get_helm.sh
-        helm lint helm/botkube
-    - name: Update and publish helm chart
-      if: success() &&  github.event_name  == 'push'  && endsWith(github.ref ,'develop')
-      run: |
-        cd ..
-        git config --global user.email "botkube-ci@infracloud.io"
-        git config --global user.name "BotKube CI"
-        git clone https://github.com/infracloudio/charts.git
-        cd charts
-        git fetch
-        git branch
-        git pull origin gh-pages
-        helm init --client-only
-        helm package --app-version v9.99.9-dev --version v9.99.9-dev ${{ github.workspace }}/helm/botkube/
-        helm repo index --url https://infracloudio.github.io/charts --merge index.yaml .
-        git add .
-        git commit -m "Update BotKube latest helm chart"
-        git push https://${{ secrets.GH_TOKEN }}@github.com/infracloudio/charts.git
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.13
+
+      - name: Install golint
+        run: |
+          go get -u golang.org/x/lint/golint
+
+      - name: Before build
+        run: |
+          echo Workflow trigger - ${{ github.event_name }}
+          ./hack/verify-gofmt.sh
+          ./hack/verify-govet.sh
+          ./hack/verify-golint.sh
+          go mod verify
+
+      - name: Build and test
+        run: |
+          make test
+          make
+
+      - name: Login to GitHub Docker Registry
+        if: success() && github.event_name == 'push' && endsWith(github.ref ,'develop')
+        run: |
+          docker login -u ${{ secrets.DOCKER_USERNAME }} -p ${{ secrets.DOCKER_PASSWORD }}
+
+      - name: Build Container Image
+        if: success() && github.event_name == 'push' && endsWith(github.ref ,'develop')
+        run: |
+          docker buildx build \
```

Comment:
Can we use `make container-image` here?

Reply:
Indeed you can however the builder sets up the environment nicely without needing to setup experimental etc..

Reply:
@xUnholy let's use the make target so that there will be no difference in a way developers test their images and the one we push. Just to avoid any surprises :)

Reply:
@xUnholy We can get this merged once this gets addressed.

Reply:
The only difference is we've changed `container-image` it would only `--load` and not push, would require more re-writing and unfortunately I'm time poor right now.